### PR TITLE
fix: sql model

### PR DESCRIPTION
**Pull request by debanjan97**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development
- [ ] Lint rules pass locally

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: models/profiles.yaml [hunk 1]
```diff
@@ -59,253 +81,6 @@ models:
...
-        - entity_var:
-            name: sources_list
-            from: models/pages_orderby_table
-            select: array_agg( context_campaign_source)
-            description: list of all sources from which a user has visited the app, sorted in chronological order, from oldest to newest
-            dependencies:
-              - session_start_time
       features:
         - days_since_last_seen
         - days_since_account_creation
```
Comment (nidhilashkari17) [inline models/profiles.yaml]: @am6010 @debanjan97 Do we not need to remove this from here if it is removed above ?

Comment (debanjan97) [reply models/profiles.yaml]: probably yes
all the removing/ adjusting was done by migration scripts, auto migration might have skipped it

Comment (debanjan97) [reply models/profiles.yaml]: will test it post removing

Comment (debanjan97) [reply models/profiles.yaml]: removed

**Pull request merged**